tests/test_connector_matrix.py

```diff
@@ -1052,3 +1066,28 @@ async def test_create_joinroom(self):
         assert event.target == "hello"
         assert event.event_id == "$143273582443PhrSn:example.org"
         assert event.raw_event == self.join_room_json
+
+    @property
+    def custom_json(self):
+        return {
+            "content": {"hello": "world"},
+            "event_id": "$15573463541827394vczPd:localhost",
+            "origin_server_ts": 1557346354253,
+            "room_id": "!test:localhost",
+            "sender": "@neo:matrix.org",
+            "type": "opsdroid.dev",
+            "unsigned": {"age": 48926251},
+            "user_id": "@nso:matrix.org",
```

Comment:
Not @neo?

Reply:
this shouldn't actually be here so I removed it and all other instances of it.